share/mk/bsd.subdir.mk

```diff
@@ -58,7 +58,7 @@ STANDALONE_SUBDIR_TARGETS+= \
 		maninstall manlint obj objlink
 
 # It is safe to install in parallel when staging.
-.if defined(NO_ROOT) || !empty(SYSROOT)
+.if (defined(NO_ROOT) || !empty(SYSROOT)) && !defined(NO_PARALLEL_INSTALL)
```

Comment:
If I understand correctly this is still doing an install in parallel but now looks at `SUBDIR_DEPEND`. Maybe something more like SUBDIR_INSTALL_USE_DEPENDS?